app/domain/errors.rb

```diff
@@ -338,6 +338,21 @@ module AuthnAzure
         code: "CONJ00054E"
       )
     end
+
+    module AuthnGcp
+
+      InvalidAudience = ::Util::TrackableErrorClass.new(
+        msg:  "'audience' token field {0} is invalid. The format should be " \
```

Comment:
@shulifink please review

Reply:
fine. I like that it tells them what to do next

Reply:
:)